readme.md

```diff
@@ -1,168 +1,182 @@
 
-### You Do: ERD (5/30)
+### You Do: ERD (5 minutes / 0:30)
+
+> 3 minutes exercise. 2 minutes review.
 
 Spend 5 minutes diagramming the ERD for Tunr.
 
-### You Do: Define the Schema(5/35)
+### You Do: Define the Schema (5 minutes / 0:35)
 
-Create the schema file in the db directory.
+Create the schema file in the db directory...
 
-```
+```bash
 $ touch db/schema.sql
 ```
 
 Using our ERD, fill in `db/schema.sql` with SQL to create the Schema. Look at the readme for a guide to the attributes of each model.
 
-### You Do: Create the DB & Load the Schema (5/40)
+<!-- AM: Include a link to SQL reference -->
 
-Create our PostgreSQL database:
+Once your schema is good to go, create the Tunr database via the command line (not `psql`)...
 
-`$ createdb tunr_db`
-
-Load the schema:
+```bash
+$ createdb tunr_db
+```
 
-`$ psql -d tunr_db < db/schema.sql`
+Then load the schema into the database, again from the command line...
 
-### You Do: Define ActiveRecord Models (10/50)
+```bash
+$ psql -d tunr_db < db/schema.sql
+```
 
-Create files for your `Artist` and `Song` classes in the `models` folder.
+### You Do: Define ActiveRecord Models (5 minutes / 0:40)
 
-Remember that the file names should be singular, to match the class name: `models/song.rb`, `models/artist.rb`
+Create files for your `Artist` and `Song` classes in the `models` folder. Remember that the file names should be singular, to match the class name (e.g., `models/song.rb`)
 
-Is there any old code that we can leverage?
+> Is there any old code that we can leverage?
 
 ### I Do: Using `console.rb`
 
 We've created a file for you called `console.rb`. Take a minute to read through it and see what it's doing.
 
-This file isn't special, we've just written it so that it loads active record, connects to the database, loads our models, and then gives us a pry console.
+This file isn't special. We've just written it so that it loads ActiveRecord, connects to the database, loads our models and allows us to interact with them using the Pry REPL.
+
+> **tl;dr** - We can use `console.rb` to test our models without loading up Sinatra.
+
+#### You Do: Play with `console.rb` (5 minutes / 0:45)
+
+Use `console.rb` to create 2 artists and 4 songs. Each song should be associated with - or "belong to" - an artist.
+
+```bash
+$ bundle exec ruby console.rb
+```
 
-We can use `console.rb` to test our AR models without loading up sinatra. It's just for playing around with them.
+<!-- AM: Do we need to use bundle exec here? -->
 
-#### You Do: Play with `console.rb` (10/60)
+## Break (10 minutes / 0:55)
 
-Use `console.rb` to create 2 artists, and 4 songs, and associate the songs with artists.
+## We Do: Load Seed Data (5 minutes / 1:00)
 
-`$ bundle exec ruby console.rb`
+Take a look at `db/seeds.rb`. We've taken some data from the iTunes API, and formatted / saved it so we can load it into our database using AR.
```

Comment:
does this have `get_artist_data` method? if so, can you refactor to just have all the seed data in a single file?